src/sentry/api/endpoints/organization_teams.py

```diff
@@ -26,11 +26,11 @@
 
 # OrganizationPermission + team:write
 class OrganizationTeamsPermission(OrganizationPermission):
+    # PUT and DELETE are irrelevant because they are handled by TeamDetailsEndpoint per the
+    # Sentry API: https://docs.sentry.io/api/teams/
     scope_map = {
-        "GET": ["org:read", "org:write", "org:admin"],
-        "POST": ["org:write", "org:admin", "team:write"],
-        "PUT": ["org:write", "org:admin", "team:write"],
-        "DELETE": ["org:admin", "team:write"],
+        "GET": ["org:read", "org:write"],
+        "POST": ["org:write", "team:write"],
```

Comment:
Note: PUT and DELETE are irrelevant because they are handled by TeamDetailsEndpoint per the
Sentry API: https://docs.sentry.io/api/teams/